framework/src/main/java/org/apache/drill/test/framework/ColumnList.java

```diff
@@ -95,20 +95,19 @@ public String toString() {
   }
 
   private boolean compare(ColumnList o1, ColumnList o2) {
-    boolean result = true;
```

Comment:
I don't see this changes for ColumnList when I cloned your repo, although I see the changes for the other two files.  Strange.